gradle.properties

```diff
@@ -4,20 +4,26 @@ org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.configuration-cache=true
 
-name=spotless
-description=Spotless - keep your code spotless with Gradle
-org=diffplug
+GROUP=com.diffplug.spotless
 
-group=com.diffplug.spotless
+SONATYPE_AUTOMATIC_RELEASE=true
+SONATYPE_HOST=DEFAULT
```

Comment:
Not sure what the possible values are, but I think we are on the `s01` host, not the default.

- s01.oss.sonatype.org

Reply:
You are right, forgot updating.